### PR TITLE
[WEB-574] chore: added new events

### PR DESCRIPTION
**Pull request by 1akhanBaheti**
### This PR includes new posthog events, and tracking based on telemetry settings.

1. Removed title, and description wherever getting captured.
2. Updated user-person-properties.
3. Refactored track-element detail logic.
4. Renamed captureEvent path parameter, to more relevant name -> routerPath.

**_Includes events for the following features:_**

1. All project settings.
2. All workspace settings.
3. Project views.
5. Project pages.
6. Issue checkout.
7. Issues layout checkout.
8. Issues filter search.
9. Issues Layout property.
10. Issues display properties.
11. Labels CRUD.
12. Comments.

Also, now tracking gets operated based on telemetry settings.

This PR is attached to issue [WEB-574](https://app.plane.so/plane/projects/02c3e1d5-d7e2-401d-a773-45ecba45d745/issues/124a0e8b-5ec1-47df-9b20-c7e5232db70e).

### CODE REVIEW

**Review by rahulramesha — COMMENTED** [review]
Most of the code is fine, could you please move the text inside the `setTractElement` into constants.

File: web/components/project/sidebar-list-item.tsx [hunk 1]
```diff
@@ -163,6 +163,7 @@ export const ProjectSidebarListItem: React.FC<Props> = observer((props) => {
     if (window.innerWidth < 768) {
       themeStore.toggleSidebar();
     }
+    setTrackElement("Project sidebar");
```
[the remark this answers is not in the document]
Comment (1akhanBaheti) [reply web/components/project/sidebar-list-item.tsx]: Currently, it's not in use, so I will remove it.

File: web/components/headers/project-view-issues.tsx [hunk 2]
```diff
@@ -282,7 +283,7 @@ export const ProjectViewIssuesHeader: React.FC = observer(() => {
         {canUserCreateIssue && (
           <Button
             onClick={() => {
-              setTrackElement("PROJECT_VIEW_PAGE_HEADER");
+              setTrackElement(E_VIEWS);
```
Comment (rahulramesha) [inline web/components/headers/project-view-issues.tsx:286]: This seems different can you check?

File: web/components/issues/sub-issues/root.tsx [hunk 3]
```diff
@@ -354,7 +356,7 @@ export const SubIssuesRoot: FC<ISubIssuesRoot> = observer((props) => {
                     <div
                       className="cursor-pointer rounded border border-custom-border-100 p-1.5 px-2 shadow transition-all hover:bg-custom-background-80"
                       onClick={() => {
-                        setTrackElement("Issue detail add sub-issue");
```
Comment (rahulramesha) [inline web/components/issues/sub-issues/root.tsx:346]: These also looks different

**Review by rahulramesha — COMMENTED** [review]
Most of the changes look good, had doubts with couple of elements can you please crosscheck and confirm?

**Review by rahulramesha — APPROVED** [review]

**Pull request closed without merging**